### PR TITLE
Contribute Swift version of the kata

### PR DESCRIPTION
**Pull request by gsl-anthonymerle**
Hi Emily,

I just watched your Youtube video regarding this series of coding katas!
I wanted to give it a try but didn't find all exercices in Swift, so here is my contribution for the TirePressure kata

Thanks,
Anthony

### CODE REVIEW

File: swift/TirePressure/Sources/TirePressure/Alarm.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public class Alarm {
+    private let lowPressureThreshold: Double = 17
+    private let highPressureThreshold: Double = 21
+
+    let _sensor = Sensor()
```
Comment (emilybache) [inline swift/TirePressure/Sources/TirePressure/Alarm.swift]: I think _sensor needs to be private like the other variables. In some language versions it's not, but I think it should be

**Comment by emilybache** [comment]
Thanks! The language versions aren't all consistent but I think the sensor does need to be private in Alarm to force you to introduce dependency injection to get it testable and not just fix it via a backdoor.

File: swift/TirePressure/Sources/TirePressure/Alarm.swift (hunk 1 above)
Comment (gsl-anthonymerle) [reply swift/TirePressure/Sources/TirePressure/Alarm.swift]: True! I've updated the line
Thanks

**Comment by emilybache** [comment]
Thanks!

**Pull request merged**